### PR TITLE
Cailey hubot

### PR DESCRIPTION
**Pull request by caggles**
add hubot folder to repo

currently:
- tells time
- creates taggable groups
- creates reminders

redis is used to save hubot's brain, need to do more testing regarding what gets save during non-graceful shutdown.

### CODE REVIEW

File: apps/hubot/scripts/example.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//    hubot what time is it? - Tells you the time
+//    hubot what's the time? - Tells you the time
+//
+module.exports = (robot) => {
+  robot.respond(/(what time is it|what's the time)/gi, (res) => {
+    const d = new Date()
+    const t = `${d.getHours()}:${d.getMinutes()} and ${d.getSeconds()} seconds`
+    res.reply(`It's ${t}`)
+  })
+}
```
Comment (ShellyXueHan) [inline apps/hubot/scripts/example.js]: missing an empty line end of file

File: apps/hubot/Procfile [hunk 2]
```diff
@@ -0,0 +1 @@
+web: bin/hubot
```
Comment (ShellyXueHan) [inline apps/hubot/Procfile]: seems that this is for heroku deployment, no need for this file.

File: apps/hubot/dev.env [hunk 3]
```diff
@@ -0,0 +1,6 @@
+ROCKETCHAT_URL_VALUE=https://chat.pathfinder.gov.bc.ca
```
Comment (ShellyXueHan) [inline apps/hubot/dev.env]: this could be a different url in dev and test environments than the production one.

File: apps/hubot/bin/hubot.cmd [hunk 4]
```diff
@@ -0,0 +1,7 @@
+@echo off
```
Comment (ShellyXueHan) [inline apps/hubot/bin/hubot.cmd]: don't need this file as it's for windows execution.

**Comment by caggles** [comment]
> Looks great! Might just need to remove/modify some files that doesn't apply when deploying on openshift. In addition, there's a deleted folder at `apps/statuspage/test/dashboard_updater/ansible/git_checkout` in this PR, is that expected?

yeah, the git_checkout thing is a bug, i talked to shea about it!

**Pull request merged**